static/_redirects

```diff
@@ -33,6 +33,9 @@
 /calico-enterprise/3.15/ /calico-enterprise/3.15/about-calico-enterprise 301
 /calico-enterprise/3.14/ /calico-enterprise/3.15/about-calico-enterprise 301
 
+# Redirects for CC pages
+/calico-cloud/latest/image-assurance/scanners/cli-based-scanner#start-the-cli-scanner /calico-cloud/latest/image-assurance/overview 301
+
```

Comment:
@bmckercher123 I also just realized there were two slightly different URLs that were being used. 

The Image Assurance page had a button called "Download Scanner", which used the URL: 
- https://docs.calicocloud.io/image-assurance/scan-image-registries#start-the-cli-scanner

I believe that is what your current redirect entry covers. 

On the other hand, the Candu URL is set to: 
- https://docs.tigera.io/calico-cloud/image-assurance/scan-image-registries

Which is a slightly different URL. 

I think we want both old URLs to redirect to the same place: 
- https://docs.tigera.io/calico-cloud/image-assurance/scanners/cli-based-scanner#start-the-cli-scanner

So I think this PR needs to have a second entry. 

I can chat in-person with you tomorrow to make sure everything makes sense.